code/__DEFINES/bitrunning.dm

```diff
@@ -0,0 +1,20 @@
+#define BITRUNNER_COST_NONE 0
+#define BITRUNNER_COST_LOW 1
+#define BITRUNNER_COST_MEDIUM 2
+#define BITRUNNER_COST_HIGH 3
+#define BITRUNNER_COST_EXTREME 20
```

Comment:
one of these things are not like the other

Reply:
intentional. some of these are domains purely for indulgence, like the cargo bike